frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -0,0 +1,276 @@
+// import { Option } from "../../../types/TagTypes";
+// import BookModal from "./BookModal";
+import DeleteModal from "./DeleteBookModal";
+import DeleteReviewModal from "./DeleteReviewModal";
+import PublishModal from "./PublishModal";
+import ReviewEditor from "./ReviewEditor";
+import SingleBook from "./SingleBook";
+
+const CreateReview = (): React.ReactElement => {
+  // const [tagsSelected, setTagsSelected] = useState<Option[]>([]);
+  // const [showBookModal, setShowBookModal] = useState<boolean>(false);
+  const [showDeleteBookModal, setShowDeleteBookModal] = useState<boolean>(
+    false,
+  );
+  const [showPublishModal, setShowPublishModal] = useState<boolean>(false);
+  const [showDeleteReviewModal, setShowDeleteReviewModal] = useState<boolean>(
+    false,
+  );
+  const [deleteBookIndex, setDeleteBookIndex] = useState<number>(-1);
+  const [books, setBooks] = useState<Book[]>([]);
+  const [review, setReview] = useState("");
+  const [featured, setFeatured] = useState("0");
+
+  // const onBookModalClose = () => setShowBookModal(false);
+  const onDeleteBookModalClose = () => setShowDeleteBookModal(false);
+  const onPublishModalClose = () => setShowPublishModal(false);
+  const onDeleteReviewModalClose = () => setShowDeleteReviewModal(false);
+
+  // const handleTagSelected = (e: Option[]) => {
+  //   setTagsSelected(e);
+  // };
+
+  // adds a book to the list of books
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const addBook = (book: Book) => {
+    setBooks([...books, book]);
+  };
+
+  // deletes a book with the given index from the list of books
+  const deleteBook = (index: number) => {
+    setBooks(books.filter((_, i) => i !== index));
+  };
+
+  // updates a book with the given index from the list of books
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const editBook = (book: Book, index: number) => {
+    deleteBook(index);
+    addBook(book);
+  };
+
+  // useEffect hook adds dummy data to the books array
+  useEffect(() => {
+    setBooks([
+      {
+        title: "Shadow Chasers",
+        coverImage:
+          "https://images-na.ssl-images-amazon.com/images/I/51WCelNAo1L.jpg",
+        titlePrefix: "The",
+        seriesOrder: "1",
+        illustrator: ["Spencer Fencer"],
+        translator: ["Translator Alligator"],
+        formats: [
+          {
+            format: "Soft cover",
+            isbn: "42069",
+            price: "13",
+          },
+        ],
+        minAge: 4,
+        maxAge: 8,
+        authors: [
+          {
+            fullName: "Sophia Spencer",
+            displayName: "Sophia Spencer",
+            attribution: "Author",
+          },
+        ],
+        publishers: [
+          {
+            fullName: "Publisher Fisher",
+            publishYear: 2012,
+          },
+        ],
+        seriesName: "The Amazing Series",
+      },
+    ]);
+  }, []);
+
+  return (
+    <Box>
+      {/* <BookModal
```

Comment:
Do we know for sure this is what the props for the BookModal will look like? Otherwise it might make more sense to remove this for now and let @liu-jeffrey add this later